### DISCUSSION TITLE
Documentation request: please add a Quick Start Guide (QSG)

### DISCUSSION THREAD

**Discussion by ayewo** [com-lihaoyi/mill#4015] [General]
As someone completely new to the Mill project, I think the project could benefit from having a QSG linked to the table of contents on https://mill-build.org/ and perhaps mentioned prominently on the landing page.

Most of the blurb on the landing page at https://mill-build.org/mill/index.html focuses on selling Mill but I wanted something immediately more hands-on: to quickly try out/install `.mill/` locally. 

Took a bit of browsing around to finally find that info here: https://mill-build.org/mill/cli/installation-ide.html#_bootstrap_scripts

**Comment by lefou** [comment]
Yeah. I also think we need to have some "Download" link. With focus on the catch-word "downlaod".

**Reply by ayewo** [reply]
Exactly. Here are 2 examples that hopefully illustrate what I have in mind.

Here's a screenshot of Bazel [homepage](https://bazel.build/).
<img width="1680" alt="image" src="https://github.com/user-attachments/assets/8359d874-5371-4ee0-b431-f2ccec7db98e">
The "Install Bazel" button highlighted in the screenshot links to an [install](https://bazel.build/install) page containing steps on how to install it for various OSes.


Here's a screenshot of the Terraform [landing page](https://www.terraform.io/).
<img width="1680" alt="image" src="https://github.com/user-attachments/assets/8cfb60d3-f00b-47ca-9304-b26e1e005bfd">
The "Download Terraform" link highlighted in the screenshot links directly to the [downloads](https://developer.hashicorp.com/terraform/downloads) page.

(I know both are examples of OSS projects backed by a large company, nonetheless, they make info on how to quickly install / download the tool super easy to scan / eye ball.) 

I would love if the Mill project can separate out this info into its own `install` or `download` page, rather than bury it in a [giant wall of text](https://mill-build.org/mill/cli/installation-ide.html#_bootstrap_scripts) as is currently the case now.

**Reply by lefou** [reply]
Completely off topic, but the Bazel page, lol. It says: "{fast, correct} -- choose two". The typical "choose two" quest has three options not just two. So the question is, what they're hiding. What's the third option: "easy"?

**Reply by ayewo** [reply]
I guess it's a pun on the common refrain: "fast, good (correct) and cheap, pick two".
Since it's a free product, cheap didn't make sense in this context, which I'm guessing is why they dropped it.

**Comment by lihaoyi** [comment]
`To begin using Mill, check out the introductory documentation for each language:` is intended to be that. It contains instructions to download the examples and run them

**Reply by ayewo** [reply]
I saw that but was expecting to see the more direct words like "Install on macOS/Linux/Windows" or "Download ..." (as @lefou) suggested.

The line you pasted is not under a prominent h2 header (nor is it indexed in the TOC on the right), making it easy to miss when a newcomer is scanning the page for actionable information.

**Reply by lihaoyi** [reply]
https://github.com/com-lihaoyi/mill/commit/2482ceed51db279cae968d0e929431bb8db6e1ff moves the getting started links a bit higher on the page
